.github/workflows/check_tz_releases.yml

```diff
@@ -25,7 +25,46 @@ jobs:
 
           if [[ $latest_tag_commit != $current_commit ]]; then 
               echo "New tz release available: ${latest_tag}"
+              new_branch="gha-bump-tz-${latest_tag}"
+              git checkout -b $new_branch
+
+              # update the submodule 
+              git -C third_party/tz checkout "${latest_tag}"
+              git add third_party/tz
+
+              # update the url in valhalla_build_timezones 
+              url_old=$(grep -oP "url=\"\Khttps://github.com/.*(?=\")" scripts/valhalla_build_timezones)
+              url_new=$(echo ${url_old} | sed -E "s/download\/[0-9]{4}[a-z]/download\/${latest_tag}/")
+
+              # first make sure the file is actually available
+
+              set +e
+              curl -Is --fail-with-body ${url_new} >/dev/null
+
+              if [[ $? -eq 0 ]]; then 
```

Comment:
if so, it's safe to replace the url in the script. Just need to replace the old tag with the new one.